CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 0.2.0 - 2023-10-30
+
+### Features
```

Comment:
I would just use a "Breaking Changes" header. We don't always include it but when there are breaking changes, that is listed as the fourth header

Reply:
Good idea!